project-koku/koku
# [COST-2475] tag based rate sql update

Fixes [COST-2475](https://issues.redhat.com/browse/COST-2475)

Changes proposed in this PR:
* For tag based usage rates: 
  * Instead of acting on each matching line item in the daily summary table, this will aggregate usage and create a new row
  * Delete existing "tag" rows before processing a date range

Testing Instructions:
1. Run standard load test customer data script on this branch
2. Ingest this data `with this other command` ...
3. Check usage cost on http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/costs/?filter[time_scope_value]=-1
4. Pull main
5. Run http://127.0.0.1:5042/api/cost-management/v1/report_data/?provider_uuid={uuid}&schema=acct10001&start_date=2022-03-01&end_date=2022-03-16 changing the dates to match curent
6. 
```
docker exec -it koku_redis bash
redis-cli
select 1
flushdb
```

7. Check usage cost on http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/costs/?filter[time_scope_value]=-1 and make sure cost is the same

---
Additional Context:
